src/renderer/assets/styles/printService.css

```diff
@@ -2,6 +2,7 @@ body .ag-render-container {
   position: fixed;
   left: -10000px;
   right: -10000px;
+  max-width: 780px;
```

Comment:
This will cause problems in landscape orientation or with custom width.

Reply:
`max-width: 100%` will restrict the PDF output to the maximal width of the editor window. A user could set a vairable width that is above this one by selecting custom page size.

Reply:
I just tested this without your patch and it's reproducible: [Untitled.pdf](https://github.com/marktext/marktext/files/7871682/Untitled.pdf). Setting a large pdf page width behaves incorrectly and the text is not fully stretched.